docs/cloud-native-security/index.rst

```diff
@@ -126,76 +122,48 @@ include: review of the customer’s runbook, summary of findings and
+Findings and Events
+-------------------
+
+Native security products produce findings (rather than raw data points). Every security finding aggregated by Security Hub includes an associated normalized severity level – Critical, High, Medium, Low, Informational.
+
+Findings that are at or above the defined minimum severity level are sent to Rackspace SIEM, where they go through correlation rules and transformed into events. Every event includes one or more correlated findings. Rackspace security specialists respond to these events.
+
+From time to time, customers may wish Rackspace to investigate a finding that is below the defined minimum severity level. Such functionality is available directly in the Security Hub user interface of the AWS console on the master account. Selecting a finding and choosing **Send to Rackspace** in the **Actions** drop-down menu will send the finding to the Rackspace SIEM to be investigated by a specialist. Standard charges apply for findings sent to Rackspace using this method for customers who opted for the per-event billing model. Please note: The **Send to Rackspace** action supports single findings only. In order to send multiple findings to Rackspace, customers should repeat the process for each finding individually.
 
 Billing
 -------
 
-Cloud Native Security has a utility billing model. The monthly charge for the
-service is calculated as the product of the number of events multiplied by
-the cost per event, or the minimum monthly fee, whichever is greater.
-
-Native security products produce findings. Findings that are at or above the
-defined minimum severity level are sent to Rackspace SIEM, where they go
-through correlation rules and transformed into events. Every event includes one
-or more correlated findings. Rackspace security specialists will respond to
-these events. The per-event fee will apply to events (SIEM-correlated
-findings). For example, if the native security products generate five findings
-that are delivered to the SIEM and, after correlation, the SIEM generates two
-events (one for two related findings and one for three related findings), the
-customer will be charged for two events only.
-
-Cloud Native Security service fees do not include the cost of the native
-security products themselves. The cost of the native products as well as the
-costs of supporting infrastructure components that are required to integrate
-the native products with the Rackspace SIEM are included in the AWS
-infrastructure charges portion of the customer's Rackspace bill.
-
-Initial deployment and configuration will be charged as a one-time
-onboarding fee.
+Cloud Native Security service fees do not include the cost of the native security products themselves. The cost of the native products as well as the costs of the infrastructure supporting the integration with the Rackspace SIEM are included in the AWS infrastructure charges portion of the customer's Rackspace bill.
```

Comment:
Cloud-Native Security service ...

Reply:
"Cloud Native Security" is the official name of the offer, so we need to keep it like that.